nautobot_ssot_aristacv/diffsync/fromcv/cloudvision.py

```diff
@@ -15,25 +16,38 @@ class CloudVision(DiffSync):
 
     top_level = ["device"]
 
+    def __init__(self, *args, job=None, **kwargs):
+        """Initialize the CloudVision DiffSync adapter."""
+        super().__init__(*args, **kwargs)
+        self.job = job
+
     def load(self):
         """Load tag data from CloudVision."""
         devices = cvutils.get_devices()
         system_tags = cvutils.get_tags_by_type(TAG.models.CREATOR_TYPE_SYSTEM)
+
         for dev in devices:
             self.device = Device(name=dev["hostname"], device_id=dev["device_id"], device_model=dev["model"])
             self.add(self.device)
 
             dev_tags = [tag for tag in cvutils.get_device_tags(device_id=dev["device_id"]) if tag in system_tags]
+
             # Check if topology_type tag exists
             list_of_tag_names = [value["label"] for value in dev_tags]
             if "topology_type" not in list_of_tag_names:
                 dev_tags.append({"label": "topology_type", "value": "-"})
 
             for tag in dev_tags:
-                if any(tag["label"] in string for string in ["hostname", "serialnumber"]):
+                if any(tag["label"] in string for string in ["hostname", "serialnumber", "Container"]):
```

Comment:
I'm a bit confused by this one. Can't you just use `if tag["label"] in ["hostname", "serialnumber", "Container"]`?

Reply:
That does work. Not sure why I made it so complicated.